Dockerfile

```diff
@@ -41,27 +41,43 @@ RUN apt-get update && apt-get install -y --allow-unauthenticated --no-install-re
 		# texlive-generic-recommended \
 		texlive-latex-extra \
 		# genome nexus
-		openjdk-8-jre && \
+		openjdk-8-jre \
+		# This is for reticulate
+		python3.8-venv && \
 	apt-get clean && \
 	rm -rf /var/lib/apt/lists/*
 
 #install pandoc 1.19.2.1 (dashboard use)
-RUN wget https://github.com/jgm/pandoc/releases/download/1.19.2.1/pandoc-1.19.2.1-1-amd64.deb
-RUN dpkg -i pandoc-1.19.2.1-1-amd64.deb	
+# RUN wget https://github.com/jgm/pandoc/releases/download/1.19.2.1/pandoc-1.19.2.1-1-amd64.deb
+# RUN dpkg -i pandoc-1.19.2.1-1-amd64.deb
+RUN wget https://github.com/jgm/pandoc/releases/download/3.0.1/pandoc-3.0.1-1-amd64.deb
+RUN dpkg -i pandoc-3.0.1-1-amd64.deb
+
+
 
 # Only copy most recent changes in code are always installed
 # Do not build from local computer
 WORKDIR /root/Genie
-COPY . .
+# Copy install packages first, because R installation takes
+# a long time and unless there are changes to the actual
+# R packages used.  So the only files copied over are
+# renv/ renv.lock and the installation R script
+COPY R/install_packages.R R/install_packages.R
+COPY renv/ renv/
+COPY renv.lock renv.lock
+RUN Rscript R/install_packages.R
 
 ENV CRYPTOGRAPHY_DONT_BUILD_RUST=true
-RUN Rscript R/install_packages.R
+COPY . .
+RUN echo "source('renv/activate.R')" >> .Rprofile
 
-RUN python3 -m pip install --no-cache-dir cython
```

Comment:
Do we not use cython anymore?

Reply:
No, Im not sure why I added this previously, but now it's not needed because it gets installed as a dependency